### PR TITLE
CI: please, go faster

### PR DESCRIPTION
**Pull request by klauer**
I'm not sure if this should be merged or if I want it to be merged, even.

That being said, I'm very frustrated with the speed of caproto's CI and this is my way of complaining.

### CODE REVIEW

File: caproto/tests/test_cli_scripts.py [hunk 1]
```diff
@@ -151,15 +191,15 @@ def test_subscribe_options(more_kwargs, ioc):
...
+def test_cli(command, args, ioc, monkeypatch):
     args = fix_arg_prefixes(ioc, args)
-    p = subprocess.Popen([sys.executable, '-um', 'caproto.tests.example_runner',
-                          '--script', command] + list(args),
-                         stdout=subprocess.PIPE, stderr=subprocess.PIPE
-                         )
-    _subprocess_communicate(p, command, timeout=10.0)
+    try:
+        call_script(monkeypatch, command, args)
+    except SystemExit as ex:
```
Comment (tacaswell) [inline caproto/tests/test_cli_scripts.py:198]: I did not know you could do this, but it makes sense...

**Review by tacaswell — APPROVED** [review]

**Comment by tacaswell** [comment]
Have you tried adding `-n4` to the pytest invocation?

**Comment by klauer** [comment]
I think CI is limited to 1 core?

**Comment by tacaswell** [comment]
There are at least 2 on travis.  I also suspect that a lot of our time is network or disk not CPU bound so over-subscribing may be worth trying.

**Comment by klauer** [comment]
I think parallelizing the CI with the changes in this PR would actually conflict (this shares the IOC)

**Comment by tacaswell** [comment]
I would prefer if the tests stay isolated enough to run in parallel, that said pytest-xdist can be pretty smart about how it splits up (ex run a module serially).

**Pull request closed without merging**